src/ion/display.rs

```diff
@@ -66,13 +70,7 @@ impl fmt::Display for Value {
                         f.write_str(", ")?
                     }
 
-                    if i.is_string() {
-                        f.write_str("\"")?;
-                        i.fmt(f)?;
-                        f.write_str("\"")?;
-                    } else {
-                        i.fmt(f)?;
-                    }
+                    i.fmt(f)?;
```

Comment:
one thing actually, how about `write!(f, "{:#}", i)`? (and the same at :94)

strings inside arrays are supposed to be always quoted and currently:
```rust
println!("{}", Value::Array(vec![Value::String("hello".into())]));
```
... will print `[ hello ]`, which can't be parsed back (making `fmt::Display` not compatible back with the parser).

this isn't a direct problem here in ion-rs (because `Value::Array` or `Value::Dictionary` always appear in the context of a `Section`, which does `{v:#}`), but I imagine someone downstream could call `value.to_string()` and get surprised 😄